vdb/lib/aqua.py

```diff
@@ -643,64 +647,70 @@ def suse_to_vuln(self, cve_data):
-                    version_start_including = ""
-                    version_end_including = ""
-                    version_start_excluding = ""
-                    version_end_excluding = version
-                    fix_version_end_including = ""
-                    fix_version_start_excluding = ""
-                    fix_version_end_excluding = ""
-                    fix_version_start_including = version
-                    if pkg_name and version:
-                        tdata = config.CVE_TPL % dict(
-                            cve_id=cve_id,
-                            cwe_id=cwe_id,
-                            assigner=assigner,
-                            references=references,
-                            description="",
-                            vectorString=vector_string,
-                            vendor="rpm",
-                            product=f"suse/{pkg_name}",
-                            version="*",
-                            edition="*",
-                            version_start_including=version_start_including,
-                            version_end_including=version_end_including,
-                            version_start_excluding=version_start_excluding,
-                            version_end_excluding=version_end_excluding,
-                            fix_version_start_including=fix_version_start_including,
-                            fix_version_end_including=fix_version_end_including,
-                            fix_version_start_excluding=fix_version_start_excluding,
-                            fix_version_end_excluding=fix_version_end_excluding,
-                            severity=severity,
-                            attackComplexity=attack_complexity,
-                            score=score,
-                            userInteraction="REQUIRED",
-                            exploitabilityScore=score,
-                            publishedDate=published_date,
-                            lastModifiedDate=last_modified_date,
-                        )
-                        try:
-                            vuln = NvdSource.convert_vuln(orjson.loads(tdata))
-                            if vuln is None:
-                                continue
-                            vuln.description = compress_str(description)
-                            ret_data.append(vuln)
-                            done_pkgs[pkg_key] = True
-                        except Exception:
-                            pass
+                    product_ids = aps.get("ProductID")
+                    if not product_ids:
+                        continue
+                    for pkg_key in product_ids:
+                        if done_pkgs.get(pkg_key):
+                            continue
+                        pkg_name, version = self.product_ref_to_name_version(pkg_key)
+                        pkg_name = pkg_name.lower().replace(" ", "-").replace(":", "/")
```

Comment:
pkg_name will also include edition and product names such as `opensuse-tumbleweed`.